src/datetime-picker.component.ts

```diff
@@ -140,6 +140,15 @@ declare var moment: any;
   `,
   styles: [
     `
+* {
```

Comment:
The * {} rule would be very questionable if it were in a global scope, but since this * {} rule is in the CSS of the component, it will be a scoped style and not override the styling anywhere else than in the datepicker. 

Do you still think we should have a more specific selector?